### ISSUE TITLE
graphql-js bindings

### ISSUE THREAD

**Issue by sgrove** [tmattio/js-bindings#2]
I'd be very curious to see what the output is, given how heavily it uses `instanceOf`, etc.!

**Comment by tmattio** [comment]
Indeed! I'm going to release a first version of the project with the bindings we need in `vscode-ocaml-platform` and `ocaml-lsp`, and happy to explore `graphql-js` after 🙂
